src/videojs.ads.js

```diff
@@ -462,9 +443,11 @@ var
             enter: function() {
               // capture current player state snapshot (playing, currentTime, src)
               this.snapshot = getPlayerSnapshot(player);
-
               // remove the poster so it doesn't flash between videos
               removeNativePoster(player);
+              // We no longer need to supress play events once an ad is playing
```

Comment:
This is the ad-playback enter handler. It seems to me that this says "do not cancel play attempts while ads are playing". That sounds good if a single video element is getting re-used for ad playback but doesn't sound good if one video is playing ads and the other is playing content. I may be misreading it.